src/Agent.Worker/JobRunner.cs

```diff
@@ -218,6 +218,12 @@ public async Task<TaskResult> RunAsync(Pipelines.AgentJobRequestMessage message,
                     }
                 }
 
+                // Expand sidecar container properties
```

Comment:
why not expand job container properties?

Reply:
It felt out of scope for what this feature is supposed to be. It can definitely be added either along with this PR or sometime in the future if its something you want for job containers, but I didnt want to add additional functionality that wasnt part of the design